presto-docs/src/main/sphinx/release/release-0.223.rst

```diff
@@ -0,0 +1,26 @@
+=============
+Release 0.223
+=============
+
+General Changes
+---------------
+
+* Fix a bug where a connector may not optimize query plan even when an optimizer is specified.
+* Support multibyte characters in ``date_format`` function.
+* Add support for using long polling for task info updates to reduce cpu load on the coordinator.
+  Set experimental.task.info-update-refresh-max-wait to enable the feature.
+
+Web UI Changes
+--------------
+
+* Fix splits timeline view.
+
+Kafka Changes
+-------------
+
+* Update Kafka version from 0.8.2.2 to 1.1.1.
+
+Drift Changes
+-------------
+
+* Updated drift to com.facebook.drift 1.19.
```

Comment:
`drift` is not something well known outside of presto developers in my opinion (as compared to kafka above). In my opinion, we are not delivering a lot of value by having this in the release notes.

Reply:
If we choose to keep this point, then change `Updated` to `Update`